src/dbnode/persist/fs/commitlog/writer.go

```diff
@@ -75,7 +75,15 @@ type commitLogWriter interface {
 		annotation ts.Annotation,
 	) error
 
-	// Flush will flush the contents to the disk, useful when first testing if first commit log is writable
+	// Sync will ensure that all writes that have been issued to the writer have been
+	// FSync'd to disk.
+	Sync() error
```

Comment:
Perhaps `FlushAndSync()` considering that's what it's doing?

Reply:
I want to stop using the word Flush entirely in the public interface (because its a little non-sensical, in this case "flush" means "flushed to chunk writer" whereas all an external caller will care about is "FSync'd to disk")

Getting rid of the flush method is outside the scope of this P.R but I'd rather not put it in this method name